<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LiveFile</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets.qrc">
    <normaloff>:/assets/icon.png</normaloff>:/assets/icon.png</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1018</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionClose_file"/>
    <addaction name="actionClose_all"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionHighlighting"/>
    <addaction name="separator"/>
    <addaction name="actionFont"/>
    <addaction name="separator"/>
    <addaction name="actionChange_Theme"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuWatcher">
    <property name="title">
     <string>Watcher</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuWatcher"/>
   <addaction name="menuOption"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/open.png</normaloff>:/assets/open.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/salvar.png</normaloff>:/assets/salvar.png</iconset>
   </property>
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose_all">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/close.png</normaloff>:/assets/close.png</iconset>
   </property>
   <property name="text">
    <string>Close All Files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F4</string>
   </property>
  </action>
  <action name="actionClose_file">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/close.png</normaloff>:/assets/close.png</iconset>
   </property>
   <property name="text">
    <string>Close File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/exit.png</normaloff>:/assets/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/icon.png</normaloff>:/assets/icon.png</iconset>
   </property>
   <property name="text">
    <string>About LiveFile</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/copy.png</normaloff>:/assets/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionHighlighting">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/highl.png</normaloff>:/assets/highl.png</iconset>
   </property>
   <property name="text">
    <string>Highlighting</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/font.png</normaloff>:/assets/font.png</iconset>
   </property>
   <property name="text">
    <string>Font...</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/find.png</normaloff>:/assets/find.png</iconset>
   </property>
   <property name="text">
    <string>Find...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/stop.png</normaloff>:/assets/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/restart.png</normaloff>:/assets/restart.png</iconset>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionChange_Theme">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/assets/theme.png</normaloff>:/assets/theme.png</iconset>
   </property>
   <property name="text">
    <string>Change Theme</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
